### ISSUE TITLE
game_template sample not working with initial setup

### ISSUE THREAD

**Issue by mco-gh** [flutter/samples#1970]
I'm on an M1 MacBook Pro running OSX 13.4.1. My flutter version is:

> 13:18:33 game_template$ flutter --version
> Flutter 3.13.0-10.0.pre.31 • channel master • https://github.com/flutter/flutter.git
> Framework • revision 285f901eba (66 minutes ago) • 2023-07-26 07:12:40 -0400
> Engine • revision ba83c144f8
> Tools • Dart 3.1.0 (build 3.1.0-348.0.dev) • DevTools 2.25.0

I clone the samples repo, cd to the game_template dir, and run "flutter run". It fails as follows:

13:15:02 game_template$ flutter run
Connected devices:
macOS (desktop) • macos  • darwin-arm64   • macOS 13.4.1 22F770820d darwin-arm64
Chrome (web)    • chrome • web-javascript • Google Chrome 115.0.5790.98

Checking for wireless devices...

[1]: macOS (macos)
[2]: Chrome (chrome)
Please choose one (or "q" to quit): 1
Resolving dependencies... 
  audioplayers 4.1.0 (5.0.0 available)
  audioplayers_android 3.0.2 (4.0.0 available)
  audioplayers_darwin 4.1.0 (5.0.0 available)
  audioplayers_linux 2.1.0 (3.0.0 available)
  audioplayers_platform_interface 5.0.1 (6.0.0 available)
  audioplayers_web 3.1.0 (4.0.0 available)
  audioplayers_windows 2.0.2 (3.0.0 available)
  collection 1.17.2 (1.18.0 available)
  file 6.1.4 (7.0.0 available)
  http 0.13.6 (1.1.0 available)
  material_color_utilities 0.5.0 (0.8.0 available)
  visibility_detector 0.3.3 (0.4.0+2 available)
Got dependencies!
Launching lib/main.dart on macOS in debug mode...
Running pod install...                                           1,982ms
2023-07-26 13:15:15.197 xcodebuild[66244:35088575] DVTCoreDeviceEnabledState: DVTCoreDeviceEnabledState_Disabled set via user default (DVTEnableCoreDevice=disabled)
--- xcodebuild: WARNING: Using the first of multiple matching destinations:
{ platform:macOS, arch:arm64, id:00006000-001411190C23401E }
{ platform:macOS, arch:x86_64, id:00006000-001411190C23401E }
/Users/marccohen/.pub-cache/hosted/pub.dev/games_services-3.0.1/macos/Classes/Leaderboards.swift:8:20: warning: 'viewState' was deprecated in macOS 11.0: Use -initWithState: instead
    viewController.viewState = .leaderboards
                   ^
/Users/marccohen/.pub-cache/hosted/pub.dev/games_services-3.0.1/macos/Classes/Leaderboards.swift:9:20: warning: 'leaderboardIdentifier' was deprecated in macOS 11.0: Use -initWithLeaderboard: instead
    viewController.leaderboardIdentifier = identifier
                   ^
/Users/marccohen/.pub-cache/hosted/pub.dev/games_services-3.0.1/macos/Classes/Leaderboards.swift:15:25: warning: 'GKScore' was deprecated in macOS 11.0: Replaced by GKLeaderboardScore
    let reportedScore = GKScore(leaderboardIdentifier: leaderboardID)
                        ^
/Users/marccohen/.pub-cache/hosted/pub.dev/games_services-3.0.1/macos/Classes/Leaderboards.swift:17:5: warning: 'GKScore' was deprecated in macOS 11.0: Replaced by GKLeaderboardScore
    GKScore.report([reportedScore]) { (error) in
    ^
/Users/marccohen/.pub-cache/hosted/pub.dev/games_services-3.0.1/macos/Classes/Achievements.swift:8:20: warning: 'viewState' was deprecated in macOS 11.0: Use -initWithState: instead
    viewController.viewState = .achievements
                   ^
/Users/marccohen/.pub-cache/hosted/pub.dev/audioplayers_darwin-4.1.0/macos/Classes/SwiftAudioplayersDarwinPlugin.swift:336:23: warning: heterogeneous collection literal could only be inferred to '[String : Any]'; add explicit type annotation if this is intentional
            eventSink(["event": "audio.onCurrentPosition", "value": millis])
                      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                                                                            as [String : Any]
/Users/marccohen/.pub-cache/hosted/pub.dev/audioplayers_darwin-4.1.0/macos/Classes/SwiftAudioplayersDarwinPlugin.swift:342:23: warning: heterogeneous collection literal could only be inferred to '[String : Any]'; add explicit type annotation if this is intentional
            eventSink(["event": "audio.onDuration", "value": millis])
                      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                                                                     as [String : Any]
/Users/marccohen/.pub-cache/hosted/pub.dev/audioplayers_darwin-4.1.0/macos/Classes/SwiftAudioplayersDarwinPlugin.swift:348:23: warning: heterogeneous collection literal could only be inferred to '[String : Any]'; add explicit type annotation if this is intentional
            eventSink(["event": "audio.onPrepared", "value": isPrepared])
                      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                                                                         as [String : Any]
/Users/marccohen/.pub-cache/hosted/pub.dev/firebase_core-2.15.0/macos/Classes/messages.g.m:199:59: warning: a function declaration without a prototype is deprecated in all versions of C [-Wstrict-prototypes]
NSObject<FlutterMessageCodec> *FirebaseCoreHostApiGetCodec() {
                                                          ^
                                                           void
/Users/marccohen/.pub-cache/hosted/pub.dev/firebase_core-2.15.0/macos/Classes/messages.g.m:279:58: warning: a function declaration without a prototype is deprecated in all versions of C [-Wstrict-prototypes]
NSObject<FlutterMessageCodec> *FirebaseAppHostApiGetCodec() {
                                                         ^
                                                          void
2 warnings generated.
/Users/marccohen/.pub-cache/hosted/pub.dev/firebase_core-2.15.0/macos/Classes/FLTFirebaseCorePlugin.m:64:42: warning: 'trackingID' is deprecated [-Wdeprecated-declarations]
  pigeonOptions.trackingId = (id)options.trackingID ?: [NSNull null];
                                         ^
In module 'FirebaseCore' imported from /Users/marccohen/.pub-cache/hosted/pub.dev/firebase_core-2.15.0/macos/Classes/FLTFirebasePlugin.h:10:
@property(nonatomic, copy, nullable) NSString *trackingID DEPRECATED_ATTRIBUTE;
                                                          ^
In module 'Darwin' imported from /Users/marccohen/.pub-cache/hosted/pub.dev/firebase_core-2.15.0/macos/Classes/FLTFirebaseCorePlugin.h:4:
        #define DEPRECATED_ATTRIBUTE        __attribute__((deprecated))
                                                           ^
/Users/marccohen/.pub-cache/hosted/pub.dev/firebase_core-2.15.0/macos/Classes/FLTFirebaseCorePlugin.m:66:47: warning: 'androidClientID' is deprecated [-Wdeprecated-declarations]
  pigeonOptions.androidClientId = (id)options.androidClientID ?: [NSNull null];
                                              ^
In module 'FirebaseCore' imported from /Users/marccohen/.pub-cache/hosted/pub.dev/firebase_core-2.15.0/macos/Classes/FLTFirebasePlugin.h:10:
@property(nonatomic, copy, nullable) NSString *androidClientID DEPRECATED_ATTRIBUTE;
                                                               ^
In module 'Darwin' imported from /Users/marccohen/.pub-cache/hosted/pub.dev/firebase_core-2.15.0/macos/Classes/FLTFirebaseCorePlugin.h:4:
        #define DEPRECATED_ATTRIBUTE        __attribute__((deprecated))
                                                           ^
/Users/marccohen/.pub-cache/hosted/pub.dev/firebase_core-2.15.0/macos/Classes/FLTFirebaseCorePlugin.m:142:13: warning: 'trackingID' is deprecated [-Wdeprecated-declarations]
    options.trackingID = initializeAppRequest.trackingId;
            ^
In module 'FirebaseCore' imported from /Users/marccohen/.pub-cache/hosted/pub.dev/firebase_core-2.15.0/macos/Classes/FLTFirebasePlugin.h:10:
@property(nonatomic, copy, nullable) NSString *trackingID DEPRECATED_ATTRIBUTE;
                                                          ^
In module 'Darwin' imported from /Users/marccohen/.pub-cache/hosted/pub.dev/firebase_core-2.15.0/macos/Classes/FLTFirebaseCorePlugin.h:4:
        #define DEPRECATED_ATTRIBUTE        __attribute__((deprecated))
                                                           ^
/Users/marccohen/.pub-cache/hosted/pub.dev/firebase_core-2.15.0/macos/Classes/FLTFirebaseCorePlugin.m:152:13: warning: 'androidClientID' is deprecated [-Wdeprecated-declarations]
    options.androidClientID = initializeAppRequest.androidClientId;
            ^
In module 'FirebaseCore' imported from /Users/marccohen/.pub-cache/hosted/pub.dev/firebase_core-2.15.0/macos/Classes/FLTFirebasePlugin.h:10:
@property(nonatomic, copy, nullable) NSString *androidClientID DEPRECATED_ATTRIBUTE;
                                                               ^
In module 'Darwin' imported from /Users/marccohen/.pub-cache/hosted/pub.dev/firebase_core-2.15.0/macos/Classes/FLTFirebaseCorePlugin.h:4:
        #define DEPRECATED_ATTRIBUTE        __attribute__((deprecated))
                                                           ^
4 warnings generated.
lib/src/audio/audio_controller.dart:155:33: Error: The type 'AppLifecycleState' is not exhaustively matched by the switch cases since it doesn't match 'AppLifecycleState.hidden'.
 - 'AppLifecycleState' is from 'dart:ui'.
Try adding a default case or cases that match 'AppLifecycleState.hidden'.
    switch (_lifecycleNotifier!.value) {
                                ^
Target kernel_snapshot failed: Exception

Command PhaseScriptExecution failed with a nonzero exit code
warning: Run script build phase 'Run Script' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase. (in target 'Flutter Assemble' from project 'Runner')
** BUILD FAILED **

Building macOS application...                                           
Exception: Build process failed
13:15:49 game_template$

**Comment by domesticmouse** [comment]
Yup, this is a known issue. The samples are built to run out of the box on the stable channel. I will have a fix for this issue that will land when stable increments to Flutter 3.13.

**Comment by filiph** [comment]
I think this might point at an underlying docs issue. People should be discouraged from starting their Flutter journey with the `master` channel. 

@mco-gh, can you tell us why you went with the `master` channel? Is it just that you like to live dangerously, or was it recommended somewhere?

The reason I'm asking is that I can imagine that if I started my Flutter journey on the bleeding edge, with many basic samples just plain not working, I might run away from Flutter pretty quickly. So I think it's important for Flutter to actively shy newcomers away from `master` or even `beta`. I realize most people choose `stable`, but it should be that 100% of newcomers choose `stable`.

Relatedly, I think all the samples here should state that they are maintained against `stable`. Maybe have a copy-paste paragraph, something like: 

> Please note that this sample assumes you are using Flutter's `stable` channel (the default). If you're using anything more bleeding edge (`beta` or `master`), you might have to do some work before the sample works on your machine.

Or something to that effect. But I still think an appeal to adventurous people to stick with `stable` for the beginning might be needed as well.

**Comment by mco-gh** [comment]
Thanks both (@domesticmouse and @filiph)!

What am I doing on the master channel? Good question - I started playing with Flutter a month or two back and don't recall how I ended up there. If I had to bet, I would guess that it was me reading somewhere that master gets you the latest and greatest goodness (at a price :). It's also possible I bumped into a problem and found some stack overflow solution saying "switch to master channel and do this...". 

At any rate, I'll switch to the stable channel and close this now. If I have any problem with that, I'll re-open this issue.

Thanks again!

**Comment by mco-gh** [comment]
Just to close the loop, changing to the stable channel, flutter upgrade, flutter clean, flutter run made this app work for me, albeit with four warnings captured below. Presumably you know about those and they're being worked on. At any rate, they weren't fatal, just messy looking.

Even with the stable channel, I'm still having a problem with Filip's tictactoe repo but I'll capture that issue in that repo.

warnings from flutter run on the game_template app:

```
13:53:11 game_template$ flutter run
Connected devices:
macOS (desktop) • macos  • darwin-arm64   • macOS 13.4.1 22F770820d darwin-arm64
Chrome (web)    • chrome • web-javascript • Google Chrome 115.0.5790.114

Checking for wireless devices...

[1]: macOS (macos)
[2]: Chrome (chrome)
Please choose one (or "q" to quit): 1
Resolving dependencies... 
< _fe_analyzer_shared 61.0.0 (was 62.0.0) (62.0.0 available)
< analyzer 5.13.0 (was 6.0.0) (6.0.0 available)
  audioplayers 4.1.0 (5.0.0 available)
  audioplayers_android 3.0.2 (4.0.0 available)
  audioplayers_darwin 4.1.0 (5.0.0 available)
  audioplayers_linux 2.1.0 (3.0.0 available)
  audioplayers_platform_interface 5.0.1 (6.0.0 available)
  audioplayers_web 3.1.0 (4.0.0 available)
  audioplayers_windows 2.0.2 (3.0.0 available)
< collection 1.17.1 (was 1.17.2) (1.18.0 available)
  file 6.1.4 (7.0.0 available)
  http 0.13.6 (1.1.0 available)
< matcher 0.12.15 (was 0.12.16) (0.12.16 available)
< material_color_utilities 0.2.0 (was 0.5.0) (0.8.0 available)
< source_span 1.9.1 (was 1.10.0) (1.10.0 available)
< stack_trace 1.11.0 (was 1.11.1) (1.11.1 available)
< stream_channel 2.1.1 (was 2.1.2) (2.1.2 available)
< test 1.24.1 (was 1.24.4) (1.24.4 available)
< test_api 0.5.1 (was 0.6.1) (0.6.1 available)
< test_core 0.5.1 (was 0.5.4) (0.5.4 available)
  visibility_detector 0.3.3 (0.4.0+2 available)
These packages are no longer being depended on:
- dart_internal 0.2.8
- web 0.1.4-beta
Changed 13 dependencies!
Launching lib/main.dart on macOS in debug mode...
Updating project for Xcode compatibility.
Upgrading project.pbxproj
Upgrading Runner.xcscheme
Running pod install...                                           2,516ms
2023-07-27 13:54:13.025 xcodebuild[58594:43971850] DVTCoreDeviceEnabledState: DVTCoreDeviceEnabledState_Disabled set via user default (DVTEnableCoreDevice=disabled)
--- xcodebuild: WARNING: Using the first of multiple matching destinations:
{ platform:macOS, arch:arm64, id:00006000-001411190C23401E }
{ platform:macOS, arch:x86_64, id:00006000-001411190C23401E }
/Users/marccohen/.pub-cache/hosted/pub.dev/games_services-3.0.1/macos/Classes/Achievements.swift:8:20: warning: 'viewState' was deprecated in macOS 11.0: Use -initWithState: instead
    viewController.viewState = .achievements
                   ^
/Users/marccohen/.pub-cache/hosted/pub.dev/games_services-3.0.1/macos/Classes/Leaderboards.swift:8:20: warning: 'viewState' was deprecated in macOS 11.0: Use -initWithState: instead
    viewController.viewState = .leaderboards
                   ^
/Users/marccohen/.pub-cache/hosted/pub.dev/games_services-3.0.1/macos/Classes/Leaderboards.swift:9:20: warning: 'leaderboardIdentifier' was deprecated in macOS 11.0: Use -initWithLeaderboard: instead
    viewController.leaderboardIdentifier = identifier
                   ^
/Users/marccohen/.pub-cache/hosted/pub.dev/games_services-3.0.1/macos/Classes/Leaderboards.swift:15:25: warning: 'GKScore' was deprecated in macOS 11.0: Replaced by GKLeaderboardScore
    let reportedScore = GKScore(leaderboardIdentifier: leaderboardID)
                        ^
/Users/marccohen/.pub-cache/hosted/pub.dev/games_services-3.0.1/macos/Classes/Leaderboards.swift:17:5: warning: 'GKScore' was deprecated in macOS 11.0: Replaced by GKLeaderboardScore
    GKScore.report([reportedScore]) { (error) in
    ^
/Users/marccohen/.pub-cache/hosted/pub.dev/audioplayers_darwin-4.1.0/macos/Classes/SwiftAudioplayersDarwinPlugin.swift:336:23: warning: heterogeneous collection literal could only be inferred to '[String : Any]'; add explicit type annotation if this is intentional
            eventSink(["event": "audio.onCurrentPosition", "value": millis])
                      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                                                                            as [String : Any]
/Users/marccohen/.pub-cache/hosted/pub.dev/audioplayers_darwin-4.1.0/macos/Classes/SwiftAudioplayersDarwinPlugin.swift:342:23: warning: heterogeneous collection literal could only be inferred to '[String : Any]'; add explicit type annotation if this is intentional
            eventSink(["event": "audio.onDuration", "value": millis])
                      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                                                                     as [String : Any]
/Users/marccohen/.pub-cache/hosted/pub.dev/audioplayers_darwin-4.1.0/macos/Classes/SwiftAudioplayersDarwinPlugin.swift:348:23: warning: heterogeneous collection literal could only be inferred to '[String : Any]'; add explicit type annotation if this is intentional
            eventSink(["event": "audio.onPrepared", "value": isPrepared])
                      ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                                                                         as [String : Any]
/Users/marccohen/.pub-cache/hosted/pub.dev/firebase_core-2.15.0/macos/Classes/messages.g.m:199:59: warning: a function declaration without a prototype is deprecated in all versions of C [-Wstrict-prototypes]
NSObject<FlutterMessageCodec> *FirebaseCoreHostApiGetCodec() {
                                                          ^
                                                           void
/Users/marccohen/.pub-cache/hosted/pub.dev/firebase_core-2.15.0/macos/Classes/messages.g.m:279:58: warning: a function declaration without a prototype is deprecated in all versions of C [-Wstrict-prototypes]
NSObject<FlutterMessageCodec> *FirebaseAppHostApiGetCodec() {
                                                         ^
                                                          void
2 warnings generated.
/Users/marccohen/.pub-cache/hosted/pub.dev/firebase_core-2.15.0/macos/Classes/FLTFirebaseCorePlugin.m:64:42: warning: 'trackingID' is deprecated [-Wdeprecated-declarations]
  pigeonOptions.trackingId = (id)options.trackingID ?: [NSNull null];
                                         ^
In module 'FirebaseCore' imported from /Users/marccohen/.pub-cache/hosted/pub.dev/firebase_core-2.15.0/macos/Classes/FLTFirebasePlugin.h:10:
@property(nonatomic, copy, nullable) NSString *trackingID DEPRECATED_ATTRIBUTE;
                                                          ^
In module 'Darwin' imported from /Users/marccohen/.pub-cache/hosted/pub.dev/firebase_core-2.15.0/macos/Classes/FLTFirebaseCorePlugin.h:4:
        #define DEPRECATED_ATTRIBUTE        __attribute__((deprecated))
                                                           ^
/Users/marccohen/.pub-cache/hosted/pub.dev/firebase_core-2.15.0/macos/Classes/FLTFirebaseCorePlugin.m:66:47: warning: 'androidClientID' is deprecated [-Wdeprecated-declarations]
  pigeonOptions.androidClientId = (id)options.androidClientID ?: [NSNull null];
                                              ^
In module 'FirebaseCore' imported from /Users/marccohen/.pub-cache/hosted/pub.dev/firebase_core-2.15.0/macos/Classes/FLTFirebasePlugin.h:10:
@property(nonatomic, copy, nullable) NSString *androidClientID DEPRECATED_ATTRIBUTE;
                                                               ^
In module 'Darwin' imported from /Users/marccohen/.pub-cache/hosted/pub.dev/firebase_core-2.15.0/macos/Classes/FLTFirebaseCorePlugin.h:4:
        #define DEPRECATED_ATTRIBUTE        __attribute__((deprecated))
                                                           ^
/Users/marccohen/.pub-cache/hosted/pub.dev/firebase_core-2.15.0/macos/Classes/FLTFirebaseCorePlugin.m:142:13: warning: 'trackingID' is deprecated [-Wdeprecated-declarations]
    options.trackingID = initializeAppRequest.trackingId;
            ^
In module 'FirebaseCore' imported from /Users/marccohen/.pub-cache/hosted/pub.dev/firebase_core-2.15.0/macos/Classes/FLTFirebasePlugin.h:10:
@property(nonatomic, copy, nullable) NSString *trackingID DEPRECATED_ATTRIBUTE;
                                                          ^
In module 'Darwin' imported from /Users/marccohen/.pub-cache/hosted/pub.dev/firebase_core-2.15.0/macos/Classes/FLTFirebaseCorePlugin.h:4:
        #define DEPRECATED_ATTRIBUTE        __attribute__((deprecated))
                                                           ^
/Users/marccohen/.pub-cache/hosted/pub.dev/firebase_core-2.15.0/macos/Classes/FLTFirebaseCorePlugin.m:152:13: warning: 'androidClientID' is deprecated [-Wdeprecated-declarations]
    options.androidClientID = initializeAppRequest.androidClientId;
            ^
In module 'FirebaseCore' imported from /Users/marccohen/.pub-cache/hosted/pub.dev/firebase_core-2.15.0/macos/Classes/FLTFirebasePlugin.h:10:
@property(nonatomic, copy, nullable) NSString *androidClientID DEPRECATED_ATTRIBUTE;
                                                               ^
In module 'Darwin' imported from /Users/marccohen/.pub-cache/hosted/pub.dev/firebase_core-2.15.0/macos/Classes/FLTFirebaseCorePlugin.h:4:
        #define DEPRECATED_ATTRIBUTE        __attribute__((deprecated))
                                                           ^
4 warnings generated.
warning: Run script build phase 'Run Script' will be run during every build because it does not specify any outputs. To address this warning, either add output dependencies to the script phase, or configure it to run in every build by unchecking "Based on dependency analysis" in the script phase. (in target 'Flutter Assemble' from project 'Runner')
Building macOS application...                                           
flutter: INFO: 2023-07-27 13:55:04.634877: main.dart: Going full screen
flutter: INFO: 2023-07-27 13:55:04.681281: AppLifecycleObserver: Subscribed to app lifecycle updates
Syncing files to device macOS...                                    92ms

Flutter run key commands.
r Hot reload. 🔥🔥🔥
R Hot restart.
h List all available interactive commands.
d Detach (terminate "flutter run" but leave application running).
c Clear the screen
q Quit (terminate the application on the device).

A Dart VM Service on macOS is available at: http://127.0.0.1:53303/xtG3ZywjI-U=/
flutter: INFO: 2023-07-27 13:55:04.952746: AudioController: Preloading sound effects
flutter: INFO: 2023-07-27 13:55:05.157318: GoRouter: Full paths for routes:
flutter:   => /
flutter:   =>   /play
flutter:   =>     /play/session/:level
flutter:   =>     /play/won
flutter:   =>   /settings
flutter: 
flutter: INFO: 2023-07-27 13:55:05.159867: GoRouter: setting initial location null
flutter: INFO: 2023-07-27 13:55:05.235716: GoRouter: Using MaterialApp configuration
flutter: INFO: 2023-07-27 13:55:05.319201: AudioController: Resuming music
flutter: INFO: 2023-07-27 13:55:05.319299: AudioController: resumeMusic() called when music is stopped. This probably means we haven't yet started the music. For example, the game was started with sound off.
flutter: INFO: 2023-07-27 13:55:05.319599: AudioController: Playing Song<Mr_Smith-Sonorus.mp3> now.
The Flutter DevTools debugger and profiler on macOS is available at:
http://127.0.0.1:9101?uri=http://127.0.0.1:53303/xtG3ZywjI-U=/
flutter: INFO: 2023-07-27 13:55:26.280291: GoRouter: pushing /settings
flutter: INFO: 2023-07-27 13:55:28.040875: GoRouter: popping /
flutter: INFO: 2023-07-27 13:55:29.207731: AudioController: Playing sound: SfxType.buttonTap
flutter: INFO: 2023-07-27 13:55:29.208086: AudioController: - Chosen filename: k2.mp3
flutter: INFO: 2023-07-27 13:55:29.208801: GoRouter: going to /play
flutter: FINE: 2023-07-27 13:55:29.239798: _InkRevealState: status: AnimationStatus.forward
flutter: FINE: 2023-07-27 13:55:29.921258: _InkRevealState: status: AnimationStatus.completed
flutter: INFO: 2023-07-27 13:55:31.865362: GoRouter: going to /
flutter: FINE: 2023-07-27 13:55:31.868629: _InkRevealState: status: AnimationStatus.reverse
flutter: FINE: 2023-07-27 13:55:32.189844: _InkRevealState: status: AnimationStatus.dismissed
flutter: INFO: 2023-07-27 13:55:36.100763: GoRouter: pushing /settings
flutter: INFO: 2023-07-27 13:55:40.516051: AudioController: Stopping music
flutter: INFO: 2023-07-27 13:55:41.222133: AudioController: Resuming music
flutter: INFO: 2023-07-27 13:55:41.222343: AudioController: Calling _musicPlayer.resume()
flutter: INFO: 2023-07-27 13:55:45.877665: GoRouter: popping /
Lost connection to device.
```

**Comment by domesticmouse** [comment]
Looking at the logs, it looks like the warnings are in https://github.com/Abedalkareem/games_services
